readme.md

```diff
@@ -909,6 +910,48 @@ Assert that `contents` does not match `regex`.
 
 Assert that `error` is falsy.
 
+## Snapshot testing
+
```

Comment:
It should also include how to handle the snapshots. Should they be committed or gitignored.